packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -568,6 +568,17 @@ class VmServiceWrapper implements VmService {
     );
     return future;
   }
+
+  @override
+  Future getInboundReferences(String isolateId, String targetId, int limit) =>
```

Comment:
nit: use `{ }` instead of `=>` as dartfmt can't really do a good job formatting these long lines as it and that matches the rest of the file.